### ISSUE TITLE
Seeing no highlight whatsoever

### ISSUE THREAD

**Issue by elisee** [wpillar/atom-trailing-spaces#16]
The latest `trailing-spaces.atom-text-editor.less` stylesheet  doesn't seem to highlight any trailing spaces at all, at least when both settings are enabled. Reverting back to just:

```less
.trailing-whitespace {
  background: #4182c4;
  border-radius: 2px;
}
```

Fixes it, so it's most likely a stylesheet issue? I'm using the One Dark theme.

**Comment by kankaristo** [comment]
Atom 0.206.0 switched to a new tiled renderer, which adds a `.tile` element between `.lines` and `.line`. This breaks the current style sheet, but should be easy to fix.

Thanks for reporting this! :+1:

**Comment by kankaristo** [comment]
Comments are "lines within lines" in Atom (a `.line` within a `.line`).

At some point there was a very good reason why the CSS had to have `.lines > .line` and not just `.lines .line`, so that it didn't apply to comments. I can't remember what that reason was, so I'm changing it to `.lines .line`... If it breaks something, I'll fix it.

This should be fixed in 0.3.1. :)
